test/units/modules/network/avi/avi_healthmonitor_test.py

```diff
@@ -0,0 +1,108 @@
+import json
+import vcr
```

Comment:
We will want to skip test if this import fails. I'll find an example you can copy.

This makes it cleaner for people running the unit tests locally without `requirements/unit.txt` installed

Reply:
```python
try:
    import vcr
except ImportError:
    raise SkipTest("vcr is needed")
```